app/editor/src/features/content/form/styled/ContentForm.tsx

```diff
@@ -1,6 +1,15 @@
 import styled from 'styled-components';
 
 export const ContentForm = styled.div`
+  #txa-summary {
```

Comment:
I always worry this type of styling will cause issues with smaller screens.  It isn't responsive.

Reply:
Updated most of it to use FieldSizes, and left height as is as I think we don't have to worry as much about vertical scroll.